pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -56,6 +57,8 @@ template <class StateVectorT, class Derived> class MeasurementsBase {
 #else
     const StateVectorT &_statevector;
 #endif
+    std::optional<std::size_t> DeviceSeed = std::nullopt;
```

Comment:
For consistency with the rest of class variables:
```suggestion
    std::optional<std::size_t> _deviceseed{std::nullopt};
```